storage-benchmark

```diff
@@ -111,7 +114,11 @@ then
     	exit 1
     else
     	echo "** Installing fio **"
-    	yum install -y fio
+      # this fails if foreman-maintain packages lock if applied
+    	# yum install -y fio
+      #
+      # alternative:
+      foreman-maintain packages install fio -y
```

Comment:
_yum install -y fio_ # this fails, if the user has applied _foreman-maintain packages lock_. This common practice on Red Hat Satellite. Red Hat support demands this way of package installation. I suggest _foreman-maintain_ instead of _satellite-maintain_, to keep compatibility with upstream.